json_schemas/agreement-details.json

```diff
@@ -0,0 +1,21 @@
+{
+  "$schema": "http://json-schema.org/schema#",
+  "additionalProperties": false,
+  "minProperties": 1,
+  "properties": {
+    "signerName": {
```

Comment:
Should name, role and user ID be required fields (cf required fields here: https://github.com/alphagov/digitalmarketplace-api/blob/master/json_schemas/users.json#L40)